src/targets/java/okhttp.js

```diff
@@ -26,17 +26,43 @@ module.exports = function (source, options) {
   code.push('OkHttpClient client = new OkHttpClient();')
     .blank()
 
-  if (source.postData.text) {
-    if (source.postData.boundary) {
-      code.push('MediaType mediaType = MediaType.parse("%s; boundary=%s");', source.postData.mimeType, source.postData.boundary)
-    } else {
-      code.push('MediaType mediaType = MediaType.parse("%s");', source.postData.mimeType)
-    }
-    code.push('RequestBody body = RequestBody.create(mediaType, %s);', JSON.stringify(source.postData.text))
+  if (source.postData.mimeType === 'multipart/form-data') {
+    code.push('RequestBody body = new MultipartBody.Builder()')
+      .push('%s.setType(MultipartBody.FORM)', opts.indent)
+
+    source.postData.params.forEach((param) => {
+      if (param.fileName) {
+        code.push('%s.addFormDataPart(%s, %s,', opts.indent, JSON.stringify(param.name), JSON.stringify(param.fileName))
+          .push('%s%sRequestBody.create(MediaType.parse("text/plain"), fileInput))', opts.indent, opts.indent)
```

Comment:
Regarding the places like here where we "interpolate" indentations into placeholders (`%s`) if I understand correctly - sure we cannot instead pass a number as first arg, like `2` in this line for example? 😃 

Meaning, could this line be like:
```js
  .push(2, 'RequestBody.create(MediaType.parse("text/plain"), fileInput))')
```

Reply:
Didn't notice, great comment! 💪🏼